backend/utils/codeValidation.js

```diff
@@ -0,0 +1,25 @@
+const ensureLegalCode = (req, res, next) => {
+  const { code } = req.body;
+  const blackList = ['throw', 'Error'];
+  const splitCode = code.split(' ');
+  let legalCode = true;
+  for (let i = 0; i < blackList.length; i += 1) {
+    if (splitCode.includes(blackList[i])) {
```

Comment:
I think you very easily could bypass this check :sweat_smile:. 
I think a regex would be ideal. (Maybe create an array of regexes to check the code against, instead of an array of words?).
Or just use `.includes` rather than split the code and check individual words several times.

Reply:
Opted to split the code to prevent false positives, e.g. `const throwing=...` triggering.

Could also use regex, I guess it would be fairly similar for each word?

I was suggested by the person who found the bug to use `const legalCode = blackList.every(keyword => code.indexOf(keyword) === -1)`, but this would not tell the user what word is illegal.